### PR TITLE
Zrecenzowane 76

### PR DESCRIPTION
**Pull request by SzymonKieczka**

### CODE REVIEW

File: backend/src/api_parsers/dblp_parser.py [hunk 1]
```diff
@@ -101,3 +107,7 @@ def _extract_affiliation(author_id: str, page: dict) -> str | None:
                 return note["text"]
         raise NoAffiliationException(author_id=author_id)
     return None
+
+
+d = DBLPParser('code smells').get_authors()
+print(len(d))
```
Comment (olgaszatkowska) [inline backend/src/api_parsers/dblp_parser.py:113]: please remove :)

File: backend/src/api_parsers/dblp_parser.py [hunk 2]
```diff
@@ -70,7 +74,9 @@ def _parse_hit_dict(self, hit: dict) -> None:
                 "publication": publication,
                 "affiliation": affiliation,
             }
-            self.authors.append(Author(**auth_data))
+            a = Author(**auth_data)
+            print(a)
```
Comment (olgaszatkowska) [inline backend/src/api_parsers/dblp_parser.py:78]: and here too

File: backend/src/api_parsers/scopus_parser.py [hunk 3]
```diff
@@ -46,6 +46,7 @@ def _parse_entry_dict(self, entry: dict) -> None:
             "title": entry["dc:title"],
             "abstract": entry.get("dc:description"),
             "citations": entry.get("citedby-count"),
+            "venue": '',
```
Comment (olgaszatkowska) [inline backend/src/api_parsers/scopus_parser.py:49]: Can we make the venue optional in model and not pass it at all?

File: backend/src/api_parsers/dblp_parser.py (hunk 1 above)
Comment (SzymonKieczka) [reply backend/src/api_parsers/dblp_parser.py:113]: my bad

File: backend/src/api_parsers/scopus_parser.py (hunk 3 above)
Comment (SzymonKieczka) [reply backend/src/api_parsers/scopus_parser.py:49]: will do in a separate task

**Review by biegasj — APPROVED** [review]

**Pull request merged**